ddl/db_integration_test.go

```diff
@@ -2328,3 +2328,17 @@ func (s *testIntegrationSuite4) TestAlterIndexVisibility(c *C) {
 	tk.MustExec("alter table t3 alter index idx invisible")
 	tk.MustQuery(query).Check(testkit.Rows("idx NO"))
 }
+
+func (s *testIntegrationSuite7) TestAutoIncrementAllocator(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	defer config.RestoreFunc()()
+	config.UpdateGlobal(func(conf *config.Config) {
+		conf.AlterPrimaryKey = false
+	})
+	tk.MustExec("drop database if exists test_create_table_option_auto_inc;")
+	tk.MustExec("create database test_create_table_option_auto_inc;")
+	tk.MustExec("use test_create_table_option_auto_inc;")
+
+	tk.MustExec("create table t (a bigint primary key) auto_increment = 10;")
+	tk.MustExec("alter table t auto_increment = 10;")
```

Comment:
Try this:
```sql
create table t (a bigint primary key) auto_increment = 10;
alter table t add column b bigint unique key auto_increment;
insert t value(1, 0);
select b from t;
alter table t auto_increment=10;
insert t value(2, 0);
select b from t;
```
The result seems right, but it deserves a trial.

Reply:
```console
tidb> alter table t add column b bigint unique key auto_increment;
(8200, "unsupported add column 'b' constraint UNIQUE KEY when altering 'test.t'")
```

Reply:
In MySQL, the result is 
```
+----+
| b  |
+----+
| 1  |
| 10 |
+----+
```

Reply:
So TiDB doesn't support add a unique key at all?